internal/test/e2e/main_test.go

```diff
@@ -23,6 +23,12 @@ var (
 	nodeName   string
 )
 
+// go1.13 compatibility cf. https://github.com/golang/go/issues/31859
+var _ = func() bool {
+	testing.Init()
```

Comment:
I think you also need to do this in the node package.

Reply:
Why? Flags aren't parsed in the node package's tests, are they?

Reply:
https://github.com/virtual-kubelet/virtual-kubelet/blob/master/node/lifecycle_test.go#L45-L53

Reply:
No call to `flag.Parse` in that snippet, even in `klog.InitFlags`, though the flags initialized by klog may never be parsed if `testing` has already parsed flags, is that what you're thinking? :confused:

Reply:
Yeah, scratch that actually.